Alpha Abyss

MilkingChicken opened the pull request:
I've done some testing, seems to be crash-free. Everything aside from animations are all done. However, I will update the music over time. The music loops as well, if you're wondering.

Commit:
Updated for Alpha Abyss.

Commit:
Deleting this because I put it in the wrong folder.

Commit:
Re=uploading this in the right place.

Commit:
Updated MapList for Alpha Abyss.

Commit:
Updated width details for AA.

Commit:
Establishing alpha.lua.

Commit:
Adding AA platforms.

Named with "-1" afterwards for future frames in the animation.

Commit:
AA theme.

I put in a placeholder (will stay if you like it) with one of my 8-bittified songs.

Commit:
AA background.

Named with "-1" after it for future frames of animation.

Commit:
Finished alpha abyss.lua

Added spawnpoints, perfect platform hitboxes, perfect positioning, all that is left is the animations for this map.

Commit:
Edited bottom-right corners for scaling.

I think I did this right, but I'm not too sure.

nthirtyone reviewed (changes requested):
Bottom and right borders, not bottom-right corner. Look into Sorona's bottom platforms.
Diagonal lines on big platform are of bad quality.
Camera is too low.

Commit:
Updated AA theme with extra Yoolips!

Commit:
Updated AA theme again.

The main instruments were too loud originally.

Commit:
Updated Penny Fox's sprites.

Commit:
Updated bottom right borders.

Commit:
Reverted back to bad Penny.

Commit:
Updated big AA platform with 3-3-3 lines.

Commit:
Updated with perfect big platform.

Commit:
New positions.

Fixed camera in accords to the player and fixed platform positions in accords to double jumps.

Commit:
Camera changes didn't actually carry over.

Fixed now.
